.travis.yml

```diff
@@ -73,6 +59,25 @@ jobs:
         - cd tests/scripts
         - ./runtest gitignore
 
+    # ========================
+    #  Optional Checks/Tests
+    # ========================
+    # - branch-history
+
+    - stage: optional
+      name: "branch-history"
```

Comment:
`branch-history` seems more accurate than `git-history` for what this script does